### ISSUE TITLE
Join table

### ISSUE THREAD

**Issue by m-kad** [go-pg/pg#189]
I have two structs

```
type Station struct {
	TableName struct{} `sql:"station"`
	ID        int      `json:"id"`
	Passkey   string   `json:"passkey" sql:"text"`
	Name      string
	Reports   []Report `json:"reports" sql:"-"`
}
type Report struct {
	TableName   struct{}  `sql:"report"`
	ID          int       `json:"id"`
	Station     Station   
	StationID   int       `json:"station"`
	AirTemp     float32   `json:"airtemp"`
	WaterTemp   float32   `json:"watertemp"`
	Pressure    float32   `json:"pressure"`
	Humidity    float32   `json:"humidity"`
	Lux         float32   `json:"lux"`
	DateMesured time.Time `json:"date"`
	Picture     []byte    `json:"picture"`
	PictureLink string    `sql:"-"`
}
```

I try to get `Report` with `Station` joined to it with
`db.Model(&report).Column("report.*", "Station").Order("date_mesured desc").First()`
as shown here https://godoc.org/gopkg.in/pg.v4#example-DB-Model-HasOne
but I get an error: `ERROR #42703 column "Station" does not exist`

Postgres log says the same 
```
ERROR:  column "Station" does not exist at character 20
STATEMENT:  SELECT "report".*, "Station" FROM "report" AS "report" ORDER BY date_mesured desc, "report"."id" LIMIT 1
LOG:  incomplete message from client
```

When I change `Station` to `station_id` I get no errors but no join either

According to README I should get join out of it. Is there a bug or am I doing something wrong. I'm using v4

My SQL
```
CREATE TABLE station(
  id serial primary key not null,
  passkey varchar(255),
  name varchar (255)
);

CREATE TABLE report(
  id bigserial primary key not null,
  station_id int not null,
  air_temp decimal(15,5),
  water_temp decimal(15,5),
  pressure decimal(15,5),
  humidity decimal(15,5),
  lux decimal(15,5),
  date_mesured timestamptz,
  picture bytea,
  constraint fk1_report_station foreign key (station_id) references station(id) match simple
  on update cascade on delete cascade
);
```

**Comment by vmihailenco** [comment]
As quick fix please put `StationID` before `Station`, e.g.

```
    StationID   int       `json:"station"`
    Station     Station  
```

I will fix go-pg properly later.

**Comment by m-kad** [comment]
Tnx, that helped.

**Comment by vmihailenco** [comment]
Also fixed this properly in v4.0.7 so the fields order does not matter.
